src/cmd-koji-upload

```diff
@@ -841,6 +853,18 @@ Environment variables are supported:
         '--reserve-id-state-file', required=False,
         help='Uses the path for a reservation file previous created')
 
+    upload_cmd.add_argument(
+        '--s3-bucket', required=False,
+        help='Store bucket information in meta.json')
+
+    upload_cmd.add_argument(
+        '--s3-key', required=False,
+        help='Store key information in meta.json')
+
+    upload_cmd.add_argument(
+        '--s3-url', required=False,
```

Comment:
Now, it's missing @miabbott's feedback in https://github.com/coreos/coreos-assembler/pull/2731/files#r834404028 :)

Reply:
Should we change the parameter name to public-url too?  I was thinking in let it as url. And only use the public-url structure .

Reply:
I think it'd be good to be consistent, but personally fine with it. I'm not really concerned about `cosa koji-upload` becoming some API that people outside the CoreOS team starts using.

Reply:
I'd prefer it to be consistent, but won't block merging on it for the same reasons Jonathan says.  :)